src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --colors-link: rgba(0, 172, 220, 1);
+}
+
 body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+header {
+  width: 100vw;
```

Comment:
Using `100vw` for width can sometimes cause horizontal scrolling issues due to the scrollbar. Consider using `100%` instead to avoid this potential issue.